### DISCUSSION TITLE
Multiline Input

### DISCUSSION THREAD

**Discussion by Dronakurl** [Textualize/textual#2696] [Q&A] [answered]
It would be great to have a multiline input. Did I miss something? Otherwise, I would maybe try to contribute such a widget.
https://stackoverflow.com/questions/76349029/how-to-get-a-multiline-input-with-python-textual-framework

**Comment by davep** [comment, marked as answer]
A `TextArea` widget is [on the roadmap](https://textual.textualize.io/roadmap/).

**Reply by Dronakurl** [reply]
Thank you, and sorry for not looking it up before asking
